.travis.yml

```diff
@@ -68,13 +68,10 @@ addons:
 language: cpp
 compiler: clang
 
-before_install:
-    - curl --retry 10 --retry-max-time 120 -L "https://launchpad.net/gcc-arm-embedded/5.0/5-2016-q2-update/+download/gcc-arm-none-eabi-5_4-2016q2-20160622-linux.tar.bz2" | tar xfj -
```

Comment:
@mikeller, https://github.com/betaflight/betaflight/pull/1111/files#diff-354f30a63fb0907d4ad57269548329e3R72 replaces this line and does the same thing, but on any platform :)